src/en/guide/testing.gdoc

```diff
@@ -96,12 +96,24 @@ This will run the @testLogin@ test in the @SimpleController@ tests. You can spec
 grails test-app some.org.* SimpleController.testLogin BookController
 {code}
 
-If you only wish to re-run failed tests use the -rerun flag
+{note}
+In Grails 2.x, adding -rerun as an argument would only run those tests which failed in the previous test-app run. This argument is no longer supported.
+{note}
+
+h4. Debugging
+
+In order to debug your tests via a remote debugger, you can add @--debug-jvm@ after @grails@ in any commands, like so:
 
 {code}
-grails test-app -rerun
+grails --debug-jvm test-app
 {code}
 
+This will the default Java remote debugging port, 5005, for you to attach a remote debugger from your editor / IDE of choice.
```

Comment:
Should have said "this will open the...", good catch!

On Tue, Dec 15, 2015, 5:32 PM graemerocher <notifications@github.com> wrote:

> In src/en/guide/testing.gdoc
> <https://github.com/grails/grails-doc/pull/406#discussion_r47713080>:
>
> >  {code}
> >
> > +This will the default Java remote debugging port, 5005, for you to attach a remote debugger from your editor / IDE of choice.
>
> grammar error here. "This will the.." I assume the "the" is not needed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grails/grails-doc/pull/406/files#r47713080>.
>